docs/en_us/configuration/server_data_conf/cluster_conf.data.md

```diff
@@ -6,69 +6,76 @@ cluster_conf.data records the cluster config.
+| Protocol              | String<br>Protocol for connect backend, support http and fcgi. Default value is http |
+| TimeoutConnSrv        | Int<br>Timeout for connect backend, in ms. Default value is 2000 |
+| TimeoutResponseHeader | Int<br>Timeout for read response header, in ms. Default value is 60000 |
+| MaxIdleConnsPerHost   | Int<br>Max idle connections to each backend per BFE. Default value is 2. |
+| MaxConnsPerHost   | Int<br>Max number of concurrent connections to each backend per BFE. 0 means no limitation. Default value is 0. |
+| RetryLevel            | Int<br>Retry level if request fail. 0: retry after connect backend fails; 1: retry after connect backend fails or forwarding GET request fails. Default value is 0. |
+| BackendConf.OutlierDetectionHttpCode            | String<br>Backend HTTP status code status detection. <br>"" means disable detection, "500" means consider "500" as backend failure and increase counter by 1. <br>Status code use format "dxx"(e.g "500"); separate multiple status code by "\|".<br>Default value is "", means disable detection. |
 | FCGIConf              | Object<br>Conf for FastCGI Protocol                |
-| FCGIConf.Root         | String<br>the root folder to the site              |
+| FCGIConf.Root         | String<br>the root folder of the website       |
 | FCGIConf.EnvVars      | Map[string]string<br>extra environment variable    |
 
 #### Health Check Config
 
 CheckConf is config of backend check.
 
-| Config Item   | Description                                                 |
-| ------------- | ----------------------------------------------------------- |
-| Schem         | String<br>Protocol for health check (HTTP/TCP)                        |
-| Uri           | String<br>Uri used in health check (HTTP)                             |
-| Host          | String<br>If check request use special host header (HTTP)             |
-| StatusCode    | Int<br>Expected response code, default value is 200 (HTTP)         |
-| FailNum       | Int<br>Unhealthy threshold (consecutive failures of check request) |
-| SuccNum       | Int<br>Healthy threshold (consecutive successes of normal request) |
-| CheckTimeout  | Int<br>Timeout for health check, in ms                             |
-| CheckInterval | Int<br>Interval of health check, in ms                             |
+| Config Item   | Description                                                  |
+| ------------- | ------------------------------------------------------------ |
+| Schem         | String<br>Protocol for health check (HTTP/TCP). Default value is http. |
+| Uri           | String<br>Uri used in health check (HTTP only). Default value is "/health_check". |
+| Host          | String<br>Host used in health check (HTTP only). Default value is "". |
+| StatusCode    | Int<br>Expected response code (HTTP only). Default value is 200. And 0 means any response code is considered expected. |
```

Comment:
And 0 means any response code is considered expected.
=> And 0 means any response code is considered valid.